RELEASE_NOTES.md

```diff
@@ -1,5 +1,10 @@
 # Git Town Release Notes
 
+## Unreleased
+
+* `git sync --all`: push tags
```

Comment:
Are we going back to documenting unreleased features in the respective PRs? (I always liked it that way)